src/vscode-cosmosdb.api.d.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export interface CosmosDBDatabase {
+    accountName: string
+    connectionString: string
+    treeItemId: string
```

Comment:
I don't think we want to return treeItemId. I think `revealTreeItem` should take in the same thing that `getDatabase` does (aka connection string for now) and the Cosmos DB extension should find the node based on that information.

As Stephen has said before - the tree item id is an internal id that we probably don't want to expose.

Reply:
As I understand @StephenWeatherford  doesn't want to store treeItemId anywhere because anytime this id can be changed and everything will be broken(Am I wrong?). But now I don't store it before extension running, so even if you changed or deleted some databases/accounts you can always run refresh and everything will be fixed(actually if in running time we catch the error that some connection got broken we can just refresh connection node).
Also, don't like this because the getDatabase command is very heavy as it goes among all attached accounts/databases and I feel like it's very not-optimal that we went through all attached databases every time when we want to reveal something.